qualtran/bloqs/factoring/rsa/__init__.py

```diff
@@ -0,0 +1,34 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+# isort:skip_file
+
+r"""Bloqs for breaking RSA cryptography systems via integer factorization.
+
+RSA cryptography is a form of public key cryptography based on the difficulty of
+factoring the product of two large prime numbers.
+
+Using RSA, the cryptographic scheme chooses two large prime numbers p, q, the product
+N, λ(n) = lcm(p - 1, q - 1), an integer e such that 1 < e < λ(n),
+and finally d as d ≡ e^-1 (mod λ(n)). The public key consists of the modulus N and the
```

Comment:
why does d get the tripple-equals and the others don't?

Reply:
I just copied from wikipedia, not gonna lie (although I'm familiar I've taken multiple classes which discussed RSA). d gets triple equals because it is congruent mod lam(n) not equal. The others are actually equal not just congruent mod lam(n).